src/pages/repo.js

```diff
@@ -4,8 +4,33 @@ import App from '../../scripts/flux/components/app'
 export default React.createClass({
   displayName: 'ModelPage',
   render () {
+    const data = {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).